### PR TITLE
fix: change old url to actual ones

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

File: _posts/2013-07-23-Back-to-the-roots.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ There it is! Let's introduce ourselves. **This is the HTTP message**.
 [w3c]: https://www.w3.org/Protocols/
 [roy]: http://roy.gbiv.com/
 [steve]: http://www.steveklabnik.com/
-[tesla]: http://docs.timdorr.apiary.io/
+[tesla]: http://timdorr.docs.apiary.io/
```
Comment (patricksmms) [inline _posts/2013-07-23-Back-to-the-roots.md]:
```suggestion
[tesla]: https://timdorr.docs.apiary.io/
```

File: _posts/2013-10-17-How-to-test-api-with-api-blueprint-and-dredd.md [hunk 2]
```diff
@@ -114,8 +114,8 @@ Dredd works with several Continuous Integration Services. I've tested it with [T
 [my previous blogpost]: http://blog.apiary.io/2013/10/10/No-more-outdated-API-documentation/
 [Readme]: https://github.com/apiaryio/dredd-example/blob/master/README.md
 [blueprint]: https://gist.github.com/netmilk/6885268
-[documentation]: http://docs.dreddexample.apiary.io/
-[mock]: http://docs.dreddexample.apiary.io/traffic
+[documentation]: https://dreddexample.docs.apiary.io/
+[mock]: https://dreddexample.docs..apiary.io/traffic
```
Comment (patricksmms) [inline _posts/2013-10-17-How-to-test-api-with-api-blueprint-and-dredd.md]:
```suggestion
[mock]: https://dreddexample.docs.apiary.io/traffic
```

File: _posts/2013-07-23-Back-to-the-roots.md (hunk 1 above)
Comment (abtris) [reply _posts/2013-07-23-Back-to-the-roots.md]: here redirect works

File: _posts/2014-10-02-RESTfest.md [hunk 3]
```diff
@@ -38,4 +38,4 @@ Feel free to comment or ask any questions.
 [RESTfest]: http://restfest.org
 [Hypermedia API "bake off"]: https://github.com/RESTFest/2014-Greenville/wiki/Hack%20Day
 [@lukestokes]: https://twitter.com/lukestokes
-[API Blueprint]: http://docs.restfest2014.apiary.io/
\ No newline at end of file
+[API Blueprint]: http://restfest2014.docs.apiary.io/
```
Comment (patricksmms) [inline _posts/2014-10-02-RESTfest.md]:
```suggestion
[API Blueprint]: https://restfest2014.docs.apiary.io/
```

**Comment by abtris** [comment]
Suggested changes committed.

File: _posts/2014-11-05-Interact-with-Apiary-from-the-Command-Line.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ published: true
 comments: true
 ---
 
-We thought we'd put together a blog post to highlight another way for you to work with Apiary, and that is through using our [Apiary CLI tool](http://client.apiary.io/). Everyone has their own workflow, and sometimes that workflow involves their favorite editor and the command line. This Apiary CLI tool can allow you to stay in that workflow while interacting directly with Apiary through our [API](http://docs.apiary.apiary.io/).
+We thought we'd put together a blog post to highlight another way for you to work with Apiary, and that is through using our [Apiary CLI tool](http://client.apiary.io/). Everyone has their own workflow, and sometimes that workflow involves their favorite editor and the command line. This Apiary CLI tool can allow you to stay in that workflow while interacting directly with Apiary through our [API](http://apiary.docs.apiary.io/).
```
Comment (patricksmms) [inline _posts/2014-11-05-Interact-with-Apiary-from-the-Command-Line.md]:
```suggestion
We thought we'd put together a blog post to highlight another way for you to work with Apiary, and that is through using our [Apiary CLI tool](http://client.apiary.io/). Everyone has their own workflow, and sometimes that workflow involves their favorite editor and the command line. This Apiary CLI tool can allow you to stay in that workflow while interacting directly with Apiary through our [API](https://apiary.docs.apiary.io/).
```

File: _posts/2015-02-17-Utilising-API-Blueprint-in-API-Clients.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ let blueprint = Blueprint(named: "blueprint.json", bundle: nil)
 
 Once you’ve loaded your blueprint, you can retrieve the different resource actions as [transitions](https://github.com/the-hypermedia-project/charter/blob/master/reference/hypermedia-elements.md#transitions).
 
-This example is going to make use of one of our example blueprints, “Polls API”. I’ve pulled out an excerpt of the example blueprint and it looks something like the following. Take a look at the [full version](http://docs.pollsapi.apiary.io/) ([source](https://github.com/apiaryio/api-blueprint/blob/be00b000e47561419f654374dd975a02083354e8/examples/Polls%20API.md)).
+This example is going to make use of one of our example blueprints, “Polls API”. I’ve pulled out an excerpt of the example blueprint and it looks something like the following. Take a look at the [full version](http://pollsapi.docs.apiary.io/) ([source](https://github.com/apiaryio/api-blueprint/blob/be00b000e47561419f654374dd975a02083354e8/examples/Polls%20API.md)).
```
Comment (patricksmms) [inline _posts/2015-02-17-Utilising-API-Blueprint-in-API-Clients.md]:
```suggestion
This example is going to make use of one of our example blueprints, “Polls API”. I’ve pulled out an excerpt of the example blueprint and it looks something like the following. Take a look at the [full version](https://pollsapi.docs.apiary.io/) ([source](https://github.com/apiaryio/api-blueprint/blob/be00b000e47561419f654374dd975a02083354e8/examples/Polls%20API.md)).
```

File: _posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md [hunk 6]
```diff
@@ -22,12 +22,12 @@ They say _"A picture is worth a thousand words"_. We think the same. This is why
 ### HTTP API
  The tutorial is using a _simplified_ Polls API version – without hypermedia controls and advanced API Blueprint features.
 
- The full version of this **simplified Polls API** can be found at <http://docs.pollsapi.apiary.io> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20API.md).
+ The full version of this **simplified Polls API** can be found at <http://pollsapi.docs.apiary.io> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20API.md).
```
Comment (patricksmms) [inline _posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md]:
```suggestion
 The full version of this **simplified Polls API** can be found at <https://pollsapi.docs.apiary.io> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20API.md).
```

File: _posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md [hunk 7]
```diff
@@ -22,12 +22,12 @@ They say _"A picture is worth a thousand words"_. We think the same. This is why
...
 
 ### REST API
 As we believe following REST constraints may bring competitive advantage in fields where API provider does not control clients we have also prepared a truly REST – _hypermedia_ – version of the Polls API.
 
-You can find the **hypermedia Polls API** at <http://docs.pollshypermedia.apiary.io/> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20Hypermedia%20API.md)
+You can find the **hypermedia Polls API** at <http://pollshypermedia.docs.apiary.io/> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20Hypermedia%20API.md)
```
Comment (patricksmms) [inline _posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md]:
```suggestion
You can find the **hypermedia Polls API** at <https://pollshypermedia.docs.apiary.io/> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20Hypermedia%20API.md)
```

**Pull request merged**